R/io.R

```diff
@@ -329,3 +329,23 @@ reclassify_raster <- function(crop_cover_levels){
                                             GCAM_Class = GCAM_commodity)
 
 }
+
+#' get_land_category
+#' @details Classify watershed condition based on percent development and cultivation.
+#' @author Kristian Nelson (kristian.nelson@pnnl.gov)
+#' @export
+get_land_category <- function(percent.area){
+
+  if(percent.area <= 1){
```

Comment:
Please use `percent_area` to keep style consistent. Check out this [style guide](https://style.tidyverse.org/syntax.html#object-names) for general reference.